windows/src/desktop/kmshell/main/Keyman.System.RemoteUpdateCheck.pas

```diff
@@ -0,0 +1,386 @@
+(*
+  Name:             WebUpdateCheck
+  Copyright:        Copyright (C) SIL International.
+  Documentation:
+  Description:
+  Create Date:      5 Dec 2023
+
+  Modified Date:
+  Authors:          rcruickshank
+  Related Files:
+  Dependencies:
+
+  Bugs:
+  Todo:
+  Notes:
+  History:
+*)
```

Comment:
Let's use the shorter modern pattern:

https://github.com/keymanapp/keyman/blob/c488460c716b4cce4ab795d8f9f772c21fcb99e0/developer/src/tike/project/Keyman.Developer.UI.Project.xmlLdmlProjectFileUI.pas#L1-L5